src/Core/Framework/Changelog/ChangelogParser.php

```diff
@@ -41,6 +42,10 @@ public function parse(SplFileInfo $file, string $rootDir): ChangelogDefinition
 
     private function findIssueIdInCommit(string $path, string $rootDir): ?string
     {
+        if (EnvironmentHelper::getVariable('CHANGELOG_DISABLE_GIT', false)) {
```

Comment:
we already have a `TESTS_RUNNING` env var, can we use that here? then we don't have to mess with the env during test execution